plugins/woocommerce-blocks/assets/js/events/event-emitter.ts

```diff
@@ -11,7 +11,7 @@ import {
 
 export type EventListener = (
 	data: unknown
-) => Promise< ObserverResponse | true > | ObserverResponse | true;
+) => Promise< ObserverResponse | boolean > | ObserverResponse | boolean;
```

Comment:
Is this because event listener callbacks can return `false` as well? I couldn't find an example of this though, is there anywhere you found this to be true?

Reply:
There was an error case that would fallback to `false` without generating a WP_Error. I think it was in `CheckoutProcessor`.